tests/test_modeling_layoutlm.py

```diff
@@ -209,36 +235,266 @@ def test_for_masked_lm(self):
         config_and_inputs = self.model_tester.prepare_config_and_inputs()
         self.model_tester.create_and_check_for_masked_lm(*config_and_inputs)
 
+    def test_for_sequence_classification(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        self.model_tester.create_and_check_for_sequence_classification(*config_and_inputs)
+
     def test_for_token_classification(self):
         config_and_inputs = self.model_tester.prepare_config_and_inputs()
         self.model_tester.create_and_check_for_token_classification(*config_and_inputs)
 
-    @cached_property
-    def big_model(self):
-        """Cached property means this code will only be executed once."""
-        checkpoint_path = "microsoft/layoutlm-large-uncased"
-        model = LayoutLMForMaskedLM.from_pretrained(checkpoint_path).to(
+
+def prepare_layoutlm_batch_inputs():
+    # Here we prepare a batch of 2 sequences to test a LayoutLM forward pass on:
+    input_ids = torch.tensor(
```

Comment:
Can we either use a shorter inputs, or add some magic comment to make black ignore the long line, because this is a waste of vertical space :-)